pkg/http/client.go

```diff
@@ -0,0 +1,160 @@
+package http
```

Comment:
is it our code? it looks like a generic code? Is there any high level wrapper of HTTP client?

Reply:
I got it from my previous project